### ISSUE TITLE
Example for flow in docs is incorrect?

### ISSUE THREAD

**Issue by juhaelee** [erikras/redux-form#3213]
For the `FieldProps` example, it looks like the flow type `FieldProps` is trying to demonstrate typing the props that get passed into the wrapped component, and not the Field itself. I couldn't find a flow type that exposed the FieldPassedProps (input & meta).

Currently looking into the code it looks like `FieldProps` is:

```
export type Props = {
  names: string[],
  component: Function | ReactClass<*>,
  format?: (value: any, name: string) => ?any,
  parse?: (value: any, name: string) => ?any,
  props?: Object,
  withRef?: boolean
}
```

when in the example its looking for something like...
```
type FieldPassedProps = {
   input: ...
   meta: ...
}
```

**Comment by rezanouri87** [comment]
Hey @juhaelee , 

I have the same problem. Have you found a way to sort this out?

**Comment by juhaelee** [comment]
i believe the `FieldPassedProps` needs to be exported from the library. Currently I just typed them myself

**Comment by juhaelee** [comment]
Looks like this was mentioned here: https://github.com/erikras/redux-form/issues/3214

**Comment by erikras** [comment]
Published fix in [`v7.0.2`](https://github.com/erikras/redux-form/releases/tag/v7.0.2).
